task/sql-tasks.js

```diff
@@ -269,7 +446,16 @@ async function task_1_20(db) {
  * @return {array}
  */
 async function task_1_21(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            OrderID as "OrderID",
+            SUM(UnitPrice * Quantity) AS "Maximum Purchase Amount, $"
+        FROM OrderDetails
+        GROUP BY OrderID
+        ORDER BY SUM(UnitPrice * Quantity) DESC

```

Comment:
why didn't you use short way (ORDER BY 2 DESC) of ORDER BY clause as in task1_4 ?